docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -165,7 +175,101 @@ The result:
 | 2019-01-01 00:00:00 | below | 29    |
 | 2020-01-01 00:00:00 | above | 22    |
 
-### Filter data by name
+### Create heatmap
+
+Use this transformation to prepare histogram data to be visualized over time. Similar to the [Heatmap panel]({{< relref "../../visualizations/heatmap" >}}), this transformation allows you to convert histogram metrics to buckets over time.
+
+#### X Bucket
+
+This setting determines how the x-axis is split into buckets.
+
+- **Size** - Specify a time interval in the input field. For example, a time range of `1h` makes the cells one hour wide on the x-axis.
+- **Count** - For non-time related series, use this option to define the number of elements in a bucket.
+
+#### Y Bucket
+
+This setting determines how the y-axis is split into buckets.
+
+#### Y Bucket scale
+
+Use this option to set the scale of the y-axes. Select from:
+
+- **Linear**
+- **Logarithmic** - Use a base 2 or base 10.
+- **Symlog** - A symmetrical logarithmic scale. Use a base 2 or base 10; allows negative values.
+
+### Extract fields
+
+Use this transformation to select one source of data and extract content from it in different formats. Set the following fields:
+
+- **Source** - Select the field for the source of data.
+- **Format** - Select one of the following:
+  - **JSON** - To parse JSON content from the source.
+  - **Key+value parse** - To parse content in the format `a=b` or `c:d` from the source.
+  - **Auto** - To discover fields automatically.
+- **Replace all fields** - Optional: Select this option if you want to hide all other fields and display only your calculated field in the visualization.
+- **Keep time** - Optional: Only available if **Replace all fields** is true. Keep the time field in the output.
+
+Consider the following data set:
+
+| timestamp           | json_data     |
```

Comment:
capitalizing the column headers where appropriate to be consistent with existing tables on this page
```suggestion
| Timestamp           | json_data     |
```